lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -484,20 +493,18 @@ $(document).ready(function() {
 
   $('.markdown-body p, .markdown-body span, .markdown-body pre, .markdown-body table').attr('dir','auto');
 
-  if( $('#wiki-history').length ){
-    var lookup = {};
+  if( $('#wiki-history').length || $('#page-history').length){
+    var options = {
+      format: 'svg',
+      size: 20,
```

Comment:
Same question. Not great to have these size dimensions be defined in so many different places.

Reply:
I see your point. I could set the size in the CSS for `img.gravatar` and `img.identicon`, but I think I would still need to enter a size option here. (Or if I leave it out, maybe it would default to a much larger size, which would be scaled back by the CSS). Or did you have another suggestion?